doc/EnvironmentVariables.md

```diff
@@ -0,0 +1,113 @@
+| StorageFolder                           | Path to place the EdgeHub database directory, defaults to **TempPath** of the current OS                                      |
+| UpstreamFanOutFactor                    | Max number of message groups to concurrently process for sending, grouped by sender, defaults to **10**                       |
+| UpstreamProtocol                        | Protocol used to for upstream connections, defaults to **Amqp**, and falls back to AmqpWs                                     |
+| UseServerHeartbeat                      | Sets the client-side heartbeat interval to 60sec for upstream AMQP connections, defaults to **true**                          |
+| experimentalFeatures                    | omit?                                                                                                                         |
+| IOTEDGE_WORKLOADURI                     | omit?                                                                                                                         |
+| IOTEDGE_APIVERSION                      | omit?                                                                                                                         |
+| IOTEDGE_MODULEGENERATIONID              | omit?                                                                                                                         |
+| EdgeHubDevServerCertificateFile         | omit?                                                                                                                         |
+| EdgeHubDevServerPrivateKeyFile          | omit?                                                                                                                         |
+| EdgeHubDevTrustBundleFile               | omit?                                                                                                                         |
+| EDGEDEVICEHOSTNAME                      | omit?                                                                                                                         |
+| IOTEDGE_IOTHUBHOSTNAME                  | omit?                                                                                                                         |
+| IOTEDGE_DEVICEID                        | omit?                                                                                                                         |
+| IOTEDGE_MODULEID                        | omit?                                                                                                                         |
+| IotHubConnectionString                  | omit?                                                                                                                         |
+| usePersistentStorage                    | obsolete?                                                                                                                     |
```

Comment:
This is needed (allows running EH with in memory storage)

Reply:
This is duplicated above with a description. The difference is this one's first char isn't capitalized.